blog/index.html

```diff
@@ -0,0 +1,337 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/ca61573632.js" crossorigin="anonymous"></script>
+    <script src="https://code.iconify.design/1/1.0.6/iconify.min.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;0,700;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="../styles/helper.css">
+    <link rel="stylesheet" href="../styles/blog.css">
+    <title>blog page</title>
+</head>
+
+<body>
+    <header class=" sticky light-mode">
+        <div id="logo"><img src="../assets/images/log.jpg" alt="logo"></div>
+        <nav class="navbar">
+            <ul>
+                <li><a href="../index.html">Home</a></li>
+                <li><a href="../about/index.html">About</a></li>
+                <li id="current-nav"><a href="#">Blog</a></li>
+                <li><a href="../contact/index.html">Contact</a></li>
+                <li><a href="../signup/index.html">Signup</a></li>
+                <li><a href="../login/index.html">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="toggle-switch">
+        <button><i class="fas fa-moon"></i></button>
+    </div>
+    <main class="articles-container">
+        <div id="article-card">
+            <div id="article-image">
+                <img src="../assets/images/undraw_web_development_w2vv.jpg">
+            </div>
+            <div id="published-time">
+                <span class="iconify" data-inline="false" data-icon="uil:calender"></span>
+                <p id="article-out">13th Sept, 2020</p>
+            </div>
+            <div id="article-content-wrapper">
+                <h4 id="article-title">Lorem ipsum dolor sit amet.</h4>
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+                    Nobis eius veniam similique debitis suscipit hic quae deleniti
+                    ratione! Nihil, ipsa minus?
+                </p>
+            </div>
+            <div id="interaction-menu">
+                <div id="reactions">
+                    <p id="nmbr-likes">10</p><i class="far fa-thumbs-up"></i>
+                </div id="reactions">
+                <div id="reactions">
+                    <p id="nmbr-shares">5</p><i class="fas fa-share"></i>
+                </div>
+                <div id="reactions">
+                    <p id="nmbr-views">10</p><i class="fas fa-eye"></i>
+                </div>
+                <div id="reactions">
+                    <p id="nmbr-comments">8</p><i class="far fa-comments"></i>
+                </div>
+                <div id="reactions">
```

Comment:
You can not have the same ID for more than one HTML element, IDs should be unique, please consider using classes if possible or unique IDs